### DISCUSSION TITLE
Menu Entry Swapper - Plugin should swap Talk and Pay-fare (so that Talk can be the default) on the cart drivers in Brimhaven and Shilo Village

### DISCUSSION THREAD

**Discussion by Speeddymon** [runelite/runelite#17300] [Ideas]
When using Pay-fare option on the cart drivers, you pay 200 coins. If you talk to the driver, you only pay 14 coins. Talk-to should be able to be configured as the default.

The pay-fare option applies to all NPCs including these 2; please give us an option to make them an exception to this checkbox:

![image](https://github.com/runelite/runelite/assets/1396056/de74826f-0d9a-486b-967f-dc2992924df1)

**Comment by YvesW** [comment]
This is not completely correct. He will ask approx 5% of the coins you have in your inventory, with a maximum of 200. Thus, talking to him is useless if you have >4k in your inventory. Additionally, if travelling by talking to him requires interacting with his dialogue/with the chatbox, which I suspect it does but have not tested, it's a pretty useless option for most players. It is a very insignificant amount of gp and you waste more time + it's more annoying. In conclusion, I suspect that most people prefer not excluding him.

From RuneLite's side, if you cannot shift+right click to put his default to ``talk to``, you can disable that config option and just manually swap the option for the other charter/travel npcs. Alternatively, custom menu swaps can probably override it.

**Reply by geheur** [reply]
The real answer is to make the custom swapper be able to override the generic swap. ez fix. And very useful in other situations.

**Reply by Speeddymon** [reply]
Yes this would be great if one could override the other.

**Reply by Speeddymon** [reply]
You do have to interact, but there's keyboard shortcuts (space bar once or twice to skip the dialogue, and then 1 to select travel to the destination point) so I don't know I agree about it being more of a waste or more annoying. More annoying for me is having to right click to talk, so I'd just like to have the option to exempt these 2 guys. The pay-fare option always takes 200 coins in my limited (done it once or twice) testing, while talking to him reduces it to the 5% of what you have on you.
